src/group_theory/group_action/option.lean

```diff
@@ -0,0 +1,53 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import group_theory.group_action.defs
+
+/-!
+# Sum instances for additive and multiplicative actions
+
+This file defines instances for additive and multiplicative actions on the binary `sum` type.
```

Comment:
This file is about `option`, not `sum`. Please also explain that `a • some b = some (a • b)` and `a • none = none`.